packages/quill/src/app/screens/ChangePassword.tsx

```diff
@@ -0,0 +1,108 @@
+import { Body, Button, InlineMessage, SubHeading } from '@mycrypto/ui';
+import { changePassword, getLoggingIn, translateRaw } from '@quill/common';
+import { Label } from '@rebass/forms/styled-components';
+import type { FormEvent, FunctionComponent } from 'react';
+import { useEffect } from 'react';
+import { useForm, yupValidator } from 'typed-react-form';
+import { object, string } from 'yup';
+
+import {
+  Box,
+  FormError,
+  FormInput,
+  FormPasswordStrength,
+  PanelBottom,
+  ScrollableContainer
+} from '@components';
+import { PASSWORD_SCHEMA } from '@screens/CreatePassword';
+import { useDispatch, useSelector } from '@store';
+import { translate } from '@translations';
+
+const SCHEMA = PASSWORD_SCHEMA.concat(
+  object({
+    currentPassword: string().required(translateRaw('PASSWORD_EMPTY'))
+  })
+);
+
+const initialValues = {
+  currentPassword: '',
+  password: '',
+  passwordConfirmation: ''
+};
+
+export const ChangePassword: FunctionComponent = () => {
+  const dispatch = useDispatch();
+  const loggingIn = useSelector(getLoggingIn);
+  const error = useSelector((state) => state.auth.error);
```

Comment:
Is there a reason this is not a selector? 🤔

Reply:
Nope